src/main/java/com/rakovets/course/java/core/practice/concurrency/thread_synchronization/skyNet/FactionThread.java

```diff
@@ -0,0 +1,40 @@
+package com.rakovets.course.java.core.practice.concurrency.thread_synchronization.skyNet;
+
+import java.util.Collections;
+import java.util.LinkedList;
+
+public class FactionThread implements Runnable {
```

Comment:
```suggestion
public class FractionThread implements Runnable {
```

Reply:
I used the word “Faction” because it represents a group, not any number of equal parts, for example, the two opposing groups that are described in the description.